blueprints/helper/class.hbs

```diff
@@ -0,0 +1,5 @@
+import { helper } from '@ember/component/helper';
+
+export default helper(function {{pascalCase name}}(positional /*, named*/) {
```

Comment:
it seems we could not use `helper` wrapper here https://rfcs.emberjs.com/id/0756-helper-default-manager/